readme.md

```diff
@@ -3,90 +3,215 @@ Penn Labs Constitution
 
 **Created**: _August 14, 2011_
 
-**Last modified**: _February 12, 2015 ([see changelog](changelog.md))_
+**Last modified**: _January 1st, 2020 ([see changelog](changelog.md))_
 
 
 ## Overview
 
-**Objective**: To improve the state of student-run technology and enable student innovation.
+**Mission Statement**: Improve the Penn community through high-quality, student-made technology and educational
+resources to enable student innovation.
 
 **Scope**: To create, maintain and distribute student-led projects that serve the needs of a vast
-population within the undergraduate community. Hire undergraduate developers to work part-time to
-achieve the stated objective.
+population within the undergraduate community. Foster and grow a community of passionate developers
+and designers to achieve the stated objective.
 
+## Commitments
 
-## Cost Breakdown
+### Diversity and Inclusion
 
-### Developers
+We recognize that the best organizations reflect a diverse collection of backgrounds and experiences. In order to
+best represent and serve the needs of the Penn community, Penn Labs should strive to cultivate a membership which reflects
+the diversity on Penn's campus.
 
-Penn Labs pays students up to $16 per hour to work on projects part-time during the school
-year. At roughly 120 hours/semester (10 hours/week over 12 weeks), this comes to **$1,920 per
-developer per semester**. These rates are in 2015 USD adjusted every three years for inflation
-by the Penn Labs Board.
+Additionally, as a well-respected organization within the student body, Penn Labs has a responsibility
+to set an example when it comes to being inclusive within our organization and promoting diversity within
+the larger Penn community. Penn Labs pledges to combat bias, support members from diverse backgrounds,
+and work to make the recruitment process as fair and inclusive as possible.
 
-The rate is consistent with rates at top research positions and the rates of computer science
-teaching assistants at Penn. Penn Labs may also, at its discretion, hire developers on a
-contract basis to fulfill specific needs not covered by the full time developers.
+### Open Source
 
-### Operations
+We believe that open source is the best way to develop cutting-edge applications while mantaining trust with
+the core user base. Open source allows us to quickly iterate using both user feedback and user code contribution.
+Perhaps more importantly, however, it allows our users to trust that we're handling their data responsibly. Specifically,
```

Comment:
Maybe add something more philosophical about why it's important to give back to the open source community? We don't just use open source, we contribute sometimes too.